src/Wordpress/DataStore.php

```diff
@@ -70,4 +71,26 @@ public function getCloudFlareEmail()
     {
         return get_option(self::EMAIL);
     }
+
+    /**
+     * @return (bool)
+     */
+    public function getIpRewrite()
+    {
+        return get_option(self::IP_REWRITE);
+    }
+
+    /**
+     * @param $value
+     *
+     * @return bool
+     */
+    public function setIpRewrite($value)
+    {
+        // Clear option with temp value
+        update_option(self::IP_REWRITE, 'cleared');
```

Comment:
This was a small hack. If there is a chance of setting the same value, `update_option` will return `false` which will return `createAPIError` in the API response. 

I removed this code.

Since the API call is toggle(for now) I believe no such thing will happen. If there is a possibility of setting the value intentional to the same value, the current code (not this one) will return `false`.